worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -234,6 +234,8 @@ function traverseDocumentForFailures(document) {
 
 function processFailures(document) {
     if (document.getFailures().isChanged()) {
+        var arguements = extractArguments(document.getRootDocument());
```

Comment:
- `arguements`!
- The `extractArguments()` function seems to internally handle the `.getRootDocument()` part.